core/src/main/scala/com/karumi/shot/domain/model.scala

```diff
@@ -1,15 +1,59 @@
 package com.karumi.shot.domain
 
+import com.karumi.shot.domain.model.{
+  FilePath,
+  ScreenshotComparisionErrors,
+  ScreenshotsSuite
+}
+
 object model {
+  type ScreenshotsSuite = Seq[Screenshot]
+  type FilePath = String
   type Folder = String
   type AppId = String
+  type ScreenshotComparisionErrors = Seq[ScreenshotComparisionError]
 }
 
 object Config {
-  val androidDependencyMode = "androidTestCompile"
-  val androidDependency = "com.facebook.testing.screenshot:core:0.4.2"
-  val screenshotsFolderName = "/screenshots"
-  val androidPluginName = "com.android.application"
-  val instrumentationTestTask = "connectedAndroidTest"
-  val packageTestApkTask = "packageDebugAndroidTest"
+  val androidDependencyMode: FilePath = "androidTestCompile"
+  val androidDependency: FilePath =
+    "com.facebook.testing.screenshot:core:0.4.2"
+  val screenshotsFolderName: FilePath = "/screenshots/"
+  val temporalScreenshotsFolder
+    : FilePath = screenshotsFolderName + "screenshots-default/"
+  val metadataFileName: FilePath = temporalScreenshotsFolder + "metadata.xml"
+  val androidPluginName: FilePath = "com.android.application"
+  val instrumentationTestTask: FilePath = "connectedAndroidTest"
+  val packageTestApkTask: FilePath = "packageDebugAndroidTest"
 }
+
+case class Screenshot(name: String,
+                      recordedScreenshotPath: String,
+                      testClass: String,
+                      testName: String,
+                      tileWidth: Int,
+                      tileHeight: Int,
+                      viewHierarchy: FilePath,
+                      absoluteFileNames: Seq[FilePath],
+                      relativeFileNames: Seq[FilePath],
+                      recordedPartsPaths: Seq[FilePath],
+                      screenshotWidth: Int,
```

Comment:
🤔 Is it possible to use the `Dimension` case class for screenshot sizes and tile sizes? Would it make it harder or easier to use?